.github/workflows/ci.yml

```
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  gatsby-for-percy:
+    name: Percy screenshots via Gatsby build
+    runs-on: ubuntu-latest
+    env:
+      PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+      - run: yarn
+      - run: yarn gatsby build
```

I've copied this file in #2020, so there might be merge conflicts, also should it be
```suggestion
      - uses: actions/setup-node@v2
         with:
           cache: 'npm'
           node-version: '14'
      - run: npm ci
      - run: npm run build
```

---

Oh lol, why are we not using yarn here... Thanks for the suggestion.